include/EVT/io/platform/f4xx/PWMf4xx.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef _EVT_PWMf4xx_
+#define _EVT_PWMf4xx_
+
+#include <stdint.h>
+
+#include <EVT/io/PWM.hpp>
+#include <EVT/io/pin.hpp>
+#include <HALf4/stm32f4xx.h>
+
+namespace EVT::core::IO {
+
+class PWMf4xx : public PWM {
+public:
+    /**
+     * Setup the given pin for PWM usage.
+     *
+     * @param pin[in] The pin to setup for PWM
+     */
+    PWMf4xx(Pin pin);
+
+    void setDutyCycle(uint32_t dutyCycle);
```

Comment:
```suggestion
    void setDutyCycle(uint32_t dutyCycle) override;
```
Add this to all the public functions